htdocs/modulebuilder/template/core/modules/mymodule/doc/pdf_standard_myobject.modules.php

```diff
@@ -263,18 +263,18 @@ public function write_file($object, $outputlangs, $srctemplatepath = '', $hidede
 		*/
 
 		//if (count($realpatharray) == 0) $this->posxpicture=$this->posxtva;
-
-		if (getMultidirOutput($object)) {
+		$dir_output = getMultidirOutput($object, $object->module) . '/' . $object->element;
+		if (!empty($dir_output)) {
```

Comment:
$dir_output can't be empty

Reply:
are you sure ? 
![image](https://github.com/user-attachments/assets/fc509d35-11c8-4c73-9ae8-1177bb590f1f)

Reply:
`$dir_output` wil be equal to `'/'.$object->element` if getMutidirOuput is empty so never empty

Reply:
please look at the screenshot, it returns null if parameters are unconsistent
@frederic34

Reply:
`$dir_output = null . '/' . $object->element;`

Reply:
???

Reply:
dialogue de sourd

Reply:
c'est pour ça que c'est épuisant d'écrire en anglais et de perdre du temps à discuter d'une PR tellement évidente ! ça freine beaucoup pour contribuer

Reply:
ok thanks to @altairis-noe who helps me understand the problem

Reply:
i hate git

Reply:
i will try to edit the code here

Reply:
I let some message on discord

Reply:
@frederic34 i've fixed the fix :) hope that phpstan and co will be ok with it

Reply:
phpstan refused